tests/net/socket/websocket/testcase.yaml

```diff
@@ -1,5 +1,8 @@
 common:
   depends_on: netif
+  # The websocket library uses int for pointers for testing and does not work
+  # reliably on 64 bit architectures.
```

Comment:
Please add a reference to a bug here.

Reply:
and 

> Fixes https://github.com/zephyrproject-rtos/zephyr/issues/56542

I do not think this fixes anything, this is just a workaround, IMO the bug shall remain open until it is fixed for real.

Reply:
Yeah good point, dropped.

Reply:
> Please add a reference to a bug here.

Added a link to the issue.